chipsec/utilcmd/txt_cmd.py

```diff
@@ -162,5 +163,10 @@ def txt_state(self):
         self._log_register("TXT_PCH_DIDVID")
         self._log_register("INSMM")
 
+    def run(self):
+        try:
+            self.func()
+        except Exception:
+            return ExitCode.ERROR
```

Comment:
Instead of returning ExitCode.Error, set self.ExitCode = ExitCode.Error. This should fix the "Explicit returns mixed with implicit returns" error.
```suggestion
            self.ExitCode = ExitCode.ERROR
```